download-build/action.yml

```
@@ -11,14 +12,33 @@ inputs:
   build-number:
     description: 'Build number'
     required: true
+  exclusions:
+    description: 'Exclude pattern from downloaded files'
+    required: false
+    default: ''
 runs:
   using: 'composite'
   steps:
     - name: Download artifacts
       shell: bash
       working-directory: ${{ inputs.local-repo-dir }}
-      run: jfrog rt download --fail-no-op --build "${{ github.event.repository.name }}/${{ inputs.build-number }}" ${{ inputs.remote-repo }}/
+      env:
+        JFROG_DL_BUILD: ${{ github.event.repository.name }}/${{ inputs.build-number }}
+        JFROG_DL_EXCLUSIONS: ${{ inputs.exclusions }}
+        JFROG_DL_REMOTE_REPO: ${{ inputs.remote-repo }}
+      run: jfrog rt download
+        --fail-no-op
+        --build "${JFROG_DL_BUILD}"
+        ${JFROG_DL_EXCLUSIONS/#?*/--exclusions "${JFROG_DL_EXCLUSIONS}"}
```

Use quotes to avoid resolution with matching local files. Also `#` seems not necessary.
```suggestion
        "${JFROG_DL_EXCLUSIONS/?*/--exclusions "${JFROG_DL_EXCLUSIONS}"}"
```